snapcraft/plugins/dotnet.py

```diff
@@ -211,15 +211,20 @@ def _get_sdk_info(self, version):
         if "sdk-linux-x64" in metadata[0]:
             # look for sdk-linux-x64 property, if it doesn't exist
             # look for ubuntu.14.04 entry as shipped during 1.1
-            sdk_packge_name = metadata[0]["sdk-linux-x64"]
+            sdk_package_url = metadata[0]["sdk-linux-x64"]
         elif "sdk-ubuntu.14.04" in metadata[0]:
-            sdk_packge_name = metadata[0]["sdk-ubuntu.14.04"]
+            sdk_package_url = metadata[0]["sdk-ubuntu.14.04"]
         else:
             raise DotNetBadReleaseDataError(version)
 
-        sdk_package_url = "{}{}".format(metadata[0]["blob-sdk"], sdk_packge_name)
+        # for older releases prepend the base sdk url
+        if not sdk_package_url.startswith("http"):
+            sdk_package_url = "{}{}".format(metadata[0]["blob-sdk"], sdk_package_url)
+
+        filename = sdk_package_url.rsplit("/", 1).pop()
```

Comment:
can you change this to use urlparse?
Something like:
```
parsed_url = parse.urlparse(sdk_package_url)
filename = os.path.basename(parse_url.path)
```
?

Reply:
sure, just pushed up that change